### DISCUSSION TITLE
Reference RPM Macros

### DISCUSSION THREAD

**Discussion by yabberyabber** [goreleaser/nfpm#663] [Q&A] [answered]
Is it possible to create an NFPM manifest that references RPM macros?

When packaging content for fedora it's generally recommended that manifests not hardcode paths like /usr/share and /usr/bin but rather reference macros like ${_sharedir} and ${_bindir}. This gets even more relevant for python and perl rpms where things can go in `${python3_sitelib}` 

Is it possible to reference these rpm macros in an nfpm manifest? I've tried it a few different ways and I generally get an install-time error.

**Comment by caarlos0** [comment, marked as answer]
its not possible.

we use https://github.com/google/rpmpack , which does not support macros either.
